nml/global_constants.py

```diff
@@ -300,6 +302,35 @@ def constant_number(name, info, pos):
     # general CB for rerandomizing
     "CB_RANDOM_TRIGGER"        : 0x01,
 
+    # station general flags
+    "STAT_FLAG_DISTRIBUTED_CARGO"     : 1,
+    "STAT_FLAG_RANDOM_ANIMATION"      : 2,
+    "STAT_FLAG_CUSTOM_FOUNDATIONS"    : 3,
+    "STAT_FLAG_EXTENDED_FOUNDATIONS"  : 4,
+
+    # station tile types
```

Comment:
If property 0E is not supported/needed, I don't think these constants are useful either.

Reply:
Only use I can see would be in `bitmask()` for `hide_wire_tiles` or `non_traversable_tiles`, so yeah probably useless.